modules/user_accounts/test/TestPlan.md

```diff
@@ -28,22 +28,22 @@ When creating or editing a user: (subtest: edit_user)
       - First name,
       - Last name,
       - Email.
-10. If password and confirmed password do not match, an error should be displayed.
+10. If password and confirmed password do not match, an error should be displayed. [Automated]
 11. Email fields containing submitted with invalid formats should generate an error. 
      
 * Ensure the confirm email text field is not displayed on the edit user page (and only on the create new user page).
 
 12. For an existing active user, edit the user's account and click 'Generate new password' and check 'Send email'.
-    Save. Check that an email is sent to the user with the new password. Check that the password rules are enforced 
```

Comment:
"Checking the password rules" is more complex since we're using Zxcvbn instead of requiring special characters. I don't feel that this needs to be done manually by user's since this is covered now by a Password value object as well as unit tests for that file.